questions/kart-course.md

```diff
@@ -0,0 +1,71 @@
+# Kart Kourse
+
+## Question
+
+Welcome to **Kart Kourse**! Here is the problem statement:
+
+You are given a list of racers' names in race position order and a list of
+position changes. Your task is to write a function that returns the new order of
+racers after applying the position changes.
+
+## Input
+
+Here is an example input:
+
+```python
+racers = ["Mario", "Luigi", "Peach", "Yoshi", "Toad"]
+position_changes = [2, 1, -1, 0, -2]
+```
+
+The first element in `position_changes` indicates that the racer in the second
```

Comment:
Elements in the position_changes should align with the elements in the racers list. Unintuitive that they don't